src/controller/database.ts

```diff
@@ -20,58 +20,70 @@ class Database {
         const entries: {[id: number]: DatabaseEntry} = {};
 
         if (fs.existsSync(path)) {
-            const rows = fs
-                .readFileSync(path, 'utf-8')
-                .split('\n')
-                .map((r) => r.trim())
-                .filter((r) => r != '');
-            for (const row of rows) {
-                const json = JSON.parse(row);
-                if (json.hasOwnProperty('id')) {
-                    entries[json.id] = json;
+            const file = fs.readFileSync(path, 'utf-8');
+
+            for (const row of file.split('\n')) {
+                if (!row) {
```

Comment:
I noticed the `r.trim()` is missing, so lines with 1 space will pass this check now.

Reply:
It should fail the `JSON.parse()` and log the error for the line (PS: `JSON.parse` also gets rid of superfluous stuff on valid lines).